client/src/index.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react'
+import React, { Component } from 'react'
```

Comment:
Do you want to make this a smart component?

Reply:
Yes! Thank you